manticore/manticore.py

```diff
@@ -612,7 +612,7 @@ def _read_memory_callback(self, state,  address, value, size):
     def _write_memory_callback(self, state, address, value, size):
         logger.debug("Write Memory %r %r %r", address, value, size)
 
-    def _decode_instruction_callback(self, state):
+    def _decode_instruction_callback(self, state, pc):
```

Comment:
Executor multiplex signals from all states. So in the forwarding it attaches the states generating the event too.